src/sentry/static/sentry/app/components/forms/selectControl.tsx

```diff
@@ -225,7 +225,7 @@ function SelectControl<OptionType extends GeneralSelectValue = GeneralSelectValu
       ...provided,
       color: '#007eff',
       backgroundColor: '#ebf5ff',
-      borderRadius: '2px',
+      borderRadius: 2,
```

Comment:
I'm assuming this is intentional, but shouldn't border radius be in px?

Reply:
Whoops this is a remnant from when I was debugging the reversed style merging.